indico/MaKaC/services/implementation/search.py

```diff
@@ -22,10 +22,10 @@
 from MaKaC.common.fossilize import fossilize
 
 from indico.modules.groups import GroupProxy
-
-#################################
-# User and group search
-#################################
+from indico.modules.events.models.events import Event
+from indico.modules.events.models.persons import EventPerson
+from indico.modules.events.util import serialize_event_person
+from indico.modules.users.util import _build_match
```

Comment:
Maybe rename it to `unaccent_match` and move it to `indico.core.db.sqlalchemy.custom.unaccent`?